Alluvial.Tests/StreamCatchupTests.cs

```diff
@@ -191,27 +134,24 @@ await StreamCatchup.Create(streamSource.UpdatedStreams())
         public async Task When_projections_are_cursors_then_catchup_does_not_replay_previously_seen_events()
         {
             var projectionStore = new InMemoryProjectionStore<BalanceProjection>();

```

Comment:
I'm failing to see how this tests this scenario. Shouldn't you run the catch up twice? Or provide the projection at an initial cursor state?